File: gherkin/java/src/main/java/gherkin/pickles/PickleTag.java
package gherkin.pickles;

public class PickleTag {
    private final PickleLocation location;
    private final String name;

    public PickleTag(PickleLocation location, String name) {
        this.location = location;
        this.name = name;
    }

    public String getName() {
        return name;
    }

}

File: gherkin/dotnet/Gherkin/Pickles/PickleTag.cs
using System.Runtime.Serialization;

namespace Gherkin.Pickles
{
    public class PickleTag
    {
        [DataMember(Name = "name")]
        public string Name { get; private set; }

        [DataMember(Name = "astNodeId")]
        public string AstNodeId { get; private set; }

        public PickleTag(string astNodeId, string name)
        {
            AstNodeId = astNodeId;
            Name = name;
        }
    }
}
